plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -919,4 +926,46 @@ public Optional<CatalogSchemaTableName> redirectTable(ConnectorSession session,
         }
         return Optional.empty();
     }
+
+    private void validateSchemaTable(SchemaTableName schemaTableName)
```

Comment:
choose a less generic name / split in two methods : schemaExists  & tableExists

Reply:
Make sense.

Reply:
Addressed! Created tableExists and schemaExists and inline the call.